content/guides/sync.md

```diff
@@ -52,14 +47,11 @@ Each component that talks to GOCDB or POEM peer authenticates itself with a host
 	DowntimesGOCDB = %(SchemaDir)s/downtimes.avsc
 	Poem = %(SchemaDir)s/metric_profiles.avsc
 	Prefilter = %(SchemaDir)s/metric_data.avsc
-	TopologyGOCDBGroupOfEndpoints = %(SchemaDir)s/group_endpoints.avsc
-	TopologyGOCDBGroupOfGroups = %(SchemaDir)s/group_groups-sites.avsc
-	TopologyGOCDBGroupOfServices = %(SchemaDir)s/group_groups-services.avsc
-	TopologyVOGroupOfEndpoints = %(SchemaDir)s/vo_group_endpoints.avsc
-	TopologyVOGroupOfGroups = %(SchemaDir)s/vo_group_groups.avsc
+	TopologyGroupOfEndpoints = %(SchemaDir)s/group_endpoints.avsc
+	TopologyGroupOfGroups = %(SchemaDir)s/group_groups.avsc
 	WeightsGstat = %(SchemaDir)s/weight_sites.avsc
 
-This section, together with a [DEFAULT] section, constitutes the full path of avro schema file for each component.
+This section, together with a `[DEFAULT]` section, constitutes the full path of avro schema file for each component. Avro schema files define the format of the data that each component is writing. `Topology*` schemas are common to `topology-gocdb-connector.py` and `topology-vo-connector.py` because there is a need of compute side to not make a difference between two topologies `Prefilter` schema is taken from `argo-egi-consumer` since `prefilter-egi.py` filters its metric results and need to write them in the same format.
```

Comment:
I'm actually referring to available options in [AvroSchemas] section so with Topology* I'm referring to two options for topology and with Prefilter I'm referring to Prefilter option in a section. Options in a section are case insensitive, but since I listed them with capitals, I'm referring to them with capitals included.